src/app/Library/CrudPanel/Traits/Relationships.php

```diff
@@ -91,32 +91,6 @@ public function getFieldsWithRelationType($relation_types): array
             ->toArray();
     }
 
-    /**
-     * Parse the field name back to the related entity after the form is submited.
-     * Its called in getAllFieldNames().
-     *
-     * @param  array  $fields
-     * @return array
-     */
-    public function parseRelationFieldNamesFromHtml($fields)
```

Comment:
This function is not longer needed since we don't change field names in crud, only in presentation, we would get back an array from the dot notation fields in form, but the field in crud would still `address.street` even if in form it is `address[street]` so we receive a nice array back.

Reply:
Lovely - we no longer go back&forth changing dot => array =>dot ❤️❤️❤️